tools/ingest-converter/src/main/java/org/logstash/ingest/JsUtil.java

```diff
@@ -0,0 +1,39 @@
+package org.logstash.ingest;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+
+final class JsUtil {
+
+    private JsUtil() {
+        // Utility Class
+    }
+
+    /**
+     * Sets up a {@link ScriptEngine} for a given file loaded after `ingest-shared.js`.
+     * @param file File to set up {@link ScriptEngine} for
+     * @return {@link ScriptEngine} for file
+     */
+    public static ScriptEngine engine(final String file) {
```

Comment:
I wonder if this should take a dir and load all the scripts from there. The way it is right now is fine, but I wonder if we should look ahead to handle the conversion of the complex pipeline like so: https://github.com/elastic/logstash/issues/7295

I'm assuming there would be `ingest-date.js`, `ingest-grok.js`, `ingest-geip.js` etc.